formbricks/formbricks
# fix: Background tiles resizing issue

[<type>fix: ](fix:) Background size resizing 
## What does this PR do?
Fix issue #2784 

-Loading the child logic jump cards with load of parent star rating card
-Manage indexes 
## How should this be tested?

Adding multiple logic jumps for star rating card
Change indexes of logic jump cards to check responsive sizing

### Required


### Appreciated


https://github.com/formbricks/formbricks/assets/18760062/9890991e-a706-423b-98e3-ac1a1de169f3
